### PR TITLE
Node: accept version 7, remove discontinued versions

### PR DESCRIPTION
**Pull request by LeBenLeBen**
Allow to use Node 7 which has been released a while ago. Remove support for Node 0.10 and 0.12 which are both [discontinued](https://github.com/nodejs/LTS#lts-schedule).

* This PR is a : Improvement

- [x] Documentation is written
- [x] [Changelog](https://github.com/liip/drifter/blob/master/CHANGELOG.md) was updated

### CODE REVIEW

**Comment by krtek4** [comment]
I wouldn't remove 0.10 and 0.12 if the repo is still available. Some older boxes / projects * might still rely on them and there are no obvious disadvantage to keep them.

* for example there was a PR some times ago to add Sf1 support ;)

**Comment by sephii** [comment]
I also think we shouldn't remove support for the older versions, as long as the packages are still available.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ This project try to follows [Semantic Versioning](http://semver.org/) since the
 
 For migration information, you can always have a look at https://liip-drifter.readthedocs.io/en/latest/migrations/.
 
+## In progress
```
Comment (sephii) [inline CHANGELOG.md]: Please replace by "Unreleased".

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ This project try to follows [Semantic Versioning](http://semver.org/) since the
...
+
+### Changed
+- Node role: add support for version 7.x, drop 0.10 and 0.12 (end-of-life)
```
Comment (sephii) [inline CHANGELOG.md]: I would put that in "Added" since it's in a way a new feature.

File: CHANGELOG.md (hunk 1 above)
Comment (LeBenLeBen) [inline CHANGELOG.md]: Done!

File: CHANGELOG.md (hunk 2 above)
Comment (LeBenLeBen) [inline CHANGELOG.md]: Done!

**Comment by sephii** [comment]
Thanks! :seedling:

**Pull request merged**